templates/rabbitmq.config.erb

```diff
@@ -109,11 +109,11 @@
   <%- end -%>
       {port, <%= @ssl_management_port %>},
       {ssl, true},
-      {ssl_opts, [<%- if @ssl_cacert %>
-                  {cacertfile, "<%= @ssl_cacert %>"},
+      {ssl_opts, [<%- if @_ssl_management_cacert %>
```

Comment:
I think regardless of how it's coded, `ssl_management_cacert` can't actually be undef, right? since it still defaults to `ssl_cacert` if it's not defined?